### ISSUE TITLE
Leaving Ubuntu client causes taskbar to appear

### ISSUE THREAD

**Issue by nmz787** [symless/synergy#5001]
I am running synergy from Win 8.1, controlling a Linux client (Ubuntu with Xubuntu window manager also installed, currently running from xubuntu) and starting a video in VLC and making it full-screen.... when I move back to the server, the xfce taskbar becomes visible. I do not have the taskbar set to autohide (meaning if VLC was not fullscreen, the taskbar is always visible)... so it appears when I leave the screen, the focus goes back to the taskbar or something else.

What can I do to fix this?

**Comment by nmz787** [comment]
Just logged out and back in using the normal Ubuntu window manager... and I don't get the Ubuntu taskbar popping up... the problem with me depending on this is Ubuntu is relatively heavier in terms of resource usage and can be noticeably more slow when I watch high-def video or something like netflix which relies on the browser.

**Comment by jrmrjnck** [comment]
I noticed the same issue with VLC in a KDE session. This is probably by design, as the preserveFocus option is provided to prevent this behavior (https://github.com/symless/synergy/wiki/Text-Config#Focus_Preservation). But, preserveFocus only works for linux clients and not linux server screens (https://github.com/symless/synergy/issues/3817). Not sure why that issue was closed - it's definitely still an issue for me.
